src/components/contentContainer/carousel.tsx

```diff
@@ -1,49 +1,75 @@
-import React, { FC, useState, Fragment, useMemo } from 'react';
+import React, { FC, useState, useMemo } from 'react';
 import { useInterval } from 'react-use';
 import styled, { css } from 'styled-components';
 
 import { ContentContainerInformationFragment } from '../../../types/graphql-types';
 import ContentfulRichText from '../contentfulRichText';
 import ButtonLink from '../elements/buttonLink';
 import Title from '../elements/title';
-import ArrowLeft from '../../icons/icon-arrow-left.svg';
-import ArrowRight from '../../icons/icon-arrow-right.svg';
 import * as colors from '../../style/colors';
 import {
   ScreenSize,
   applyMediaQueryMd,
   applyMediaQueryLg,
   contentMargin,
-  contentMaxWidth,
 } from '../../style/dimensions';
+import Paragraph from '../elements/paragraph';
+import { formatDateRange } from '../../formatHelpers';
+import { useIntl } from 'gatsby-plugin-intl';
 
 interface Props {
   data: ContentContainerInformationFragment;
 }
 
-const carouselHeight: Record<ScreenSize, string> = {
-  sm: '335px',
-  md: '380px',
-  lg: '380px',
+const slideHeight: Record<ScreenSize, string> = {
```

Comment:
Would it make sense to define all dimension constants as numbers? Maybe at least the new ones?